### DISCUSSION TITLE
Welcome to material-shell Discussions!

### DISCUSSION THREAD

**Discussion by PapyElGringo** [material-shell/material-shell#515] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ragnarokxg** [comment]
Would you want some help with development.  I am looking to get back into development part time, and even though I am just beginning to learn TypeScript, I would be willing to help wherever I can.

**Comment by fugbixer** [comment]
your funding seems be quite inadequate for the amount of work , @PapyElGringo would you be okay to add crypto donation addresses? best for me would be XMR and for everyone else in terms of privacy. Sadly i neither have currently the time nor skills to help out.

**Comment by PapyElGringo** [comment]
@fugbixer Thanks for willing to help funding the project as it's indeed very difficult to find some funds! I added a new section in the readme with some wallet addresses https://github.com/material-shell/material-shell/blob/main/README.md#crypto-funding

Thanks you in advaaaaance !

**Comment by fugbixer** [comment]
@PapyElGringo  support comes without demands, thank you very much for the huge amount of work so far.

>it's indeed very difficult to find some funds! 

most people on earth are poor making people who contribute foss hero's in terms of helping to push society out of the dust.
governments are poor as well, of course not in terms of resources but of in terms of reliability.

Personally had to restructure some business efforts since many people are still blind and don't see goodness of giving back. 
It's always sad but everyone needs to carry his own package first and i don't judge people for that. 
A free market of ideas is beneficial for everyone and I'm glad you took part in it.

**Reply by PapyElGringo** [reply]
Thanks mate for your support 🎉 👍
